src/CognitiveServices/CognitiveServices/help/Add-AzCognitiveServicesAccountNetworkRule.md

```diff
@@ -0,0 +1,234 @@
+Add IpRules or VirtualNetworkRules to the NetworkRule property of a Cognitive Services account
+
+## SYNTAX
+
+### NetWorkRuleString (Default)
+```
+Add-AzCognitiveServicesAccountNetworkRule [-ResourceGroupName] <String> [-Name] <String>
+ -VirtualNetworkResourceId <String[]> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### IpRuleObject
+```
+Add-AzCognitiveServicesAccountNetworkRule [-ResourceGroupName] <String> [-Name] <String> -IpRule <PSIpRule[]>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### NetworkRuleObject
+```
+Add-AzCognitiveServicesAccountNetworkRule [-ResourceGroupName] <String> [-Name] <String>
+ -VirtualNetworkRule <PSVirtualNetworkRule[]> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### IpRuleString
+```
+Add-AzCognitiveServicesAccountNetworkRule [-ResourceGroupName] <String> [-Name] <String>
+ -IpAddressOrRange <String[]> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Add-AzCognitiveServicesAccountNetworkRule** cmdlet adds IpRules or VirtualNetworkRules to the NetworkRule property of a Cognitive Services account
+
+## EXAMPLES
+
+### Example 1: Add several IpRules with IpAddressOrRange
+```
+PS C:\>Add-AzCognitiveServicesAccountNetworkRule -ResourceGroupName "myResourceGroup" -Name "myaccount" -IpAddressOrRange "200.0.0.0/24","28.2.0.0/16"
+```
+
+This command add several IpRules with IpAddressOrRange.
+
+### Example 2: Add a VirtualNetworkRule with VirtualNetworkResourceID
+```
+PS C:\>$subnet = Get-AzVirtualNetwork -ResourceGroupName "myResourceGroup" -Name "myvirtualnetwork" | Get-AzVirtualNetworkSubnetConfig
+PS C:\>Add-AzCognitiveServicesAccountNetworkRule -ResourceGroupName "myResourceGroup" -Name "myaccount" -VirtualNetworkResourceId $subnet[0].Id
+```
+
+This command add a VirtualNetworkRule with VirtualNetworkResourceID.
+
+### Example 3: Add VirtualNetworkRules with VirtualNetworkRule Objects from another account
+```
+PS C:\> $networkrule = Get-AzCognitiveServicesAccountNetworkRuleSet -ResourceGroupName "myResourceGroup" -Name "myaccount1"
+PS C:\> Add-AzCognitiveServicesAccountNetworkRule -ResourceGroupName "myResourceGroup" -Name "myaccount2" -VirtualNetworkRule $networkrule.VirtualNetworkRules
+```
+
+This command add VirtualNetworkRules with VirtualNetworkRule Objects from another account.
+
+### Example 4: Add several IpRule with IpRule objects, input with JSON
+```
+PS C:\>Add-AzCognitiveServicesAccountNetworkRule -ResourceGroupName "myResourceGroup" -Name "myaccount" -IpRule (@{IpAddressOrRange="200.0.0.0/24"},@{IpAddressOrRange="28.2.0.0/16"})
+```
+
+This command add several IpRule with IpRule objects, input with JSON.
+
+## PARAMETERS
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: IAzureContextContainer
```

Comment:
@yangyuan the build error is due to an intermittent error we're seeing on ADO, so pushing another commit through should resolve the issue. One change that you should make is to update your markdown help files to include the full type names where they appear. This should just require adding the `-UseFullTypeName` parameter when calling `Update-MarkdownHelpModule` in `platyPS`.